engine/unity5/Assets/Scripts/GUI/ToolbarStates/EmulationToolbarState.cs

```diff
@@ -44,13 +52,39 @@ public override void Start()
             Button helpButton = Auxiliary.FindObject(helpMenu, "CloseHelpButton").GetComponent<Button>();
             helpButton.onClick.RemoveAllListeners();
             helpButton.onClick.AddListener(CloseHelpMenu);
+
+            EditorApplication.wantsToQuit += BoutaQuit;
```

Comment:
Not able to compile a build with this. Errors that is "does not exist in the current context."